CHANGELOG.md

```diff
@@ -105,6 +105,7 @@
 
 #### Jsonnet
 
+* [9836](https://github.com/grafana/loki/pull/9836) **manohar-koukuntla**: Fix for TSDB config with jsonnet.
```

Comment:
This entry is not required, because your previous change has not been released yet.